.github/workflows/dockerhub-description.yml

```diff
@@ -19,3 +19,39 @@ jobs:
           repository: yuzutech/kroki
           short-description: ${{ github.event.repository.description }}
           readme-filepath: ./DOCKERHUB.md
+
+      - name: Docker Hub Description - bpmn
+        uses: peter-evans/dockerhub-description@v2.4.2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: yuzutech/kroki-bpmn
+          short-description: Companion image for BPMN to use with kroki server
```

Comment:
I tend to prefer this sentence structure: "Companion server for Kroki that provides X".